Swiftfin/Components/HTMLFormattedText.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import SwiftUI
+
+struct HTMLFormattedText: UIViewRepresentable {
+    let text: String
+    private let textView = UITextView()
+
+    init(_ content: String) {
+        self.text = content
+    }
+
+    func makeUIView(context: UIViewRepresentableContext<Self>) -> UITextView {
+        textView.widthAnchor.constraint(equalToConstant: UIScreen.main.bounds.width).isActive = true
+        textView.isSelectable = false
+        textView.isUserInteractionEnabled = false
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.isScrollEnabled = false
+        return textView
+    }
+
+    func updateUIView(_ uiView: UITextView, context: UIViewRepresentableContext<Self>) {
+        DispatchQueue.main.async {
+            if let attributeText = self.converHTML(text: text) {
+                textView.attributedText = attributeText
+            } else {
+                textView.text = ""
+            }
+        }
+    }
+
+    private func converHTML(text: String) -> NSAttributedString? {
```

Comment:
We will have to determine whether the text has HTML and then "render" it as HTML.